src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1224,6 +1224,7 @@ Color.white, new Procedure("condition4"),
 			item.inventorySize = 10;
 			item.inventoryStackSize = 42;
 			item.guiBoundTo = emptyLists || guis.isEmpty() ? null : getRandomItem(random, guis);
+            item.openGUIOnRightClick = new LogicProcedure(item.guiBoundTo == null ? null : "condition3", _true);

```

Comment:
Ideally this condition would use e.g. _true flag to test one case of fixed bool value (procedure name null) and one case of procedural condition